charts/kserve-resources/templates/clusterservingruntimes.yaml

```diff
@@ -421,6 +421,14 @@ spec:
         limits:
           cpu: "1"
           memory: 2Gi
+      volumeMounts:
+      - name: devshm
+        mountPath: /dev/shm
+  volumes:
```

Comment:
We can add a flag to enable injecting the devshm volume and volume mount

Reply:
added a flag to enable the devsh mount. however I set the default value false and set the default sizeLimit to empty as well. As I found the value can be very high from the Knative IPC issue. If user want to enable with default value, then they can enable in the helmchart. Otherwise they can mount directly in the isvc as well.

Reply:
Tested with 10GB which works for llama3 70B model using tensor parallelism 8

Reply:
Hi @yuzisun, do you expect a default sizeLimit being set here? and what value you proposed.